boardcfg/ethernet_sk2/app_settings.h

```diff
@@ -141,6 +141,13 @@ extern "C" {
 #define RDM_RESPONDER_MUTE_PORT_BIT PORTS_BIT_POS_12
 
 /**
+ * @brief The pin to use for the RDM temperature sensor.
+ *
+ * If undefined, we use a 'dummy' sensor that generates random values.
```

Comment:
But I mean I don't think it's possible to tell if the sensor is real or not if it always acts the same, perhaps switch it to amps or something if it's not a genuine reading?